### PR TITLE
refactor(pick-list, pick-list-item, pick-list-group): relocate base :host styles

### PR DESCRIPTION
**Pull request by rmstinson**
**Related Issue:** #2454 #2453 

## Summary

Updated the base :host styles for the component. This affects pick-lists & value lists. Value list pr to follow.

### CODE REVIEW

File: .husky/pre-commit
Comment (jcfranco) [inline .husky/pre-commit:5]: Can you merge the latest from `master`? This is unrelated and should not be committed.

File: package.json
Comment (jcfranco) [inline package.json:56]: These changes are unrelated and should not be committed.

**Review by jcfranco — CHANGES_REQUESTED** [review]

**Comment by bstifle** [comment]
@rmstinson yo!

so, theres not enough in the demo page to give this an accurate review.

Can you add tip, tip-group and some iterations of those?

**Comment by bstifle** [comment]
@rmstinson lookin good man! approved from my end

File: .husky/pre-commit
Comment (julio8a) [reply .husky/pre-commit:5]: @justinhough, can you uncommit this?

Comment (julio8a) [reply .husky/pre-commit:5]: @justinhough, can you uncommitted this?

Comment (justinhough) [reply .husky/pre-commit]: @julio8a Just removed it in the latest commit.

File: package.json
Comment (julio8a) [inline package.json:153]: This seems outside of the pick list refactor

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss
Comment (asangma) [inline src/components/calcite-pick-list-item/calcite-pick-list-item.scss:4]: This should either be `text-color-1` or not set since we're setting it on title and description.

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss
Comment (asangma) [inline src/components/calcite-pick-list-item/calcite-pick-list-item.scss:13]: Use the `@apply` here.

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss
Comment (asangma) [inline src/components/calcite-pick-list-item/calcite-pick-list-item.scss:14]: Use `@apply` here.

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss
Comment (asangma) [inline src/components/calcite-pick-list-item/calcite-pick-list-item.scss:72]: Use `@apply` here. You'll have to split it into two rules.

File: src/components/calcite-pick-list-group/calcite-pick-list-group.scss
Comment (asangma) [inline src/components/calcite-pick-list-group/calcite-pick-list-group.scss:7]: Use `@apply` here.

File: src/components/calcite-pick-list/calcite-pick-list.scss
Comment (asangma) [inline src/components/calcite-pick-list/calcite-pick-list.scss:13]: Use `@apply` here.

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss
Comment (asangma) [inline src/components/calcite-pick-list-item/calcite-pick-list-item.scss:76]: We should add `font-normal` to ensure this gets the correct font weight.

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss
Comment (asangma) [inline src/components/calcite-pick-list-item/calcite-pick-list-item.scss:80]: We should add `font-normal` to ensure this gets the correct font weight.

**Review by asangma — CHANGES_REQUESTED** [review]
A few minor tweaks. :)

File: src/components/calcite-pick-list-group/calcite-pick-list-group.scss
Comment (asangma) [inline src/components/calcite-pick-list-group/calcite-pick-list-group.scss:2]: Include `box-border` on :host.

File: src/components/calcite-pick-list-item/calcite-pick-list-item.scss
Comment (asangma) [inline src/components/calcite-pick-list-item/calcite-pick-list-item.scss:2]: Include `box-border` on :host.

File: src/components/calcite-pick-list/calcite-pick-list.scss
Comment (asangma) [inline src/components/calcite-pick-list/calcite-pick-list.scss:2]: Include `box-border` on :host.

**Review by asangma — CHANGES_REQUESTED** [review]

**Comment by julio8a** [comment]
@asangma, can you give it another look?

File: src/components/calcite-pick-list/calcite-pick-list.scss
Comment (asangma) [inline src/components/calcite-pick-list/calcite-pick-list.scss:11]: This is missing the wildcard selector and box-border style from the component-host partial.

**Review by asangma — CHANGES_REQUESTED** [review]
Nice!
Just two really minor fixes.

**Comment by asangma** [comment]
Looks like the value-list PR (#2775) has some overlapping updates on pick-list, pick-list-item, and pick-list-goup.
It might be a good idea to redo these updates in that PR.

File: src/components/calcite-pick-list-group/calcite-pick-list-group.scss
Comment (julio8a) [reply src/components/calcite-pick-list-group/calcite-pick-list-group.scss:2]: @justinhough, can you add the wildcard selector for the box-border that was here before?

File: src/components/calcite-pick-list/calcite-pick-list.scss
Comment (julio8a) [reply src/components/calcite-pick-list/calcite-pick-list.scss:11]: @justinhough, this one too (add the wild card selector and box-border)

**Review by julio8a — APPROVED** [review]

**Comment by asangma** [comment]
@justinhough This has some overlap with #2775. Perhaps we should merge that PR first and then handle conflicts here?

**Review by asangma — APPROVED** [review]
This looks good but I think we should merge #2775 first

**Review by asangma — CHANGES_REQUESTED** [review]
Blocking this until #2775 is merged.

**Comment by justinhough** [comment]
@asangma Merged in master with PR #2775 included.

cc @julio8a

**Review by justinhough — COMMENTED** [review]
Done

**Pull request merged**
